### PR TITLE
Update domain lisk.io to lisk.com

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

- The lisk domain was changed from lisk.io to lisk.com, which was not reflected in codebase

### How was it solved?

- Updated all the domain references

### How was it tested?

- Click on new domain links and check if they navigate to correct page

### CODE REVIEW

**Review by MichalTuleja — COMMENTED** [review]
There's still quite a few spots with `.io` suffix. I need to confirm with DevOps which are already updated before updating them and merging.

**Comment by ManuGowda** [comment]
> There's still quite a few spots with `.io` suffix. I need to confirm with DevOps which are already updated before updating them and merging.

Let me know the missing places, I will update them 👍🏼

**Comment by MichalTuleja** [comment]
@ManuGowda I've updated the docs & configs, can you please have a final look?

File: README.md [hunk 1]
```diff
@@ -4,27 +4,30 @@
...
+The main focus of Lisk Service is to provide data to the UI clients such as Lisk Desktop and Lisk Mobile. Lisk Service makes it possible to access all live blockchain data in a similar manner to the regular Lisk SDK API. 
+In addition, Lisk Service also provides users with much more detailed information and endpoints, such as geolocation and various statistics about network usage, etc.
 
 The project implementation is based on Microservices. The technical stack is designed to deliver several micro-services, and each of them provides one particular functionality. The data is served in JSON format and exposed by a public RESTful API.
+
 ## Available Services
 
-Lisk Service consists of several separate modules, that can be run independently from the others. Gateway is required to expose the APIs provided by particular services.
+Lisk Service consists of several separate modules, that can be run independently from the others. The Gateway is required to expose the APIs provided by the specific services.
```
Comment (sameersubudhi) [inline README.md:14]:
```suggestion
Lisk Service consists of several separate modules, that can be run independently from the others. The Gateway is required to expose the APIs provided by the specific services via HTTP and WS protocols.
```

File: README.md [hunk 2]
```diff
@@ -76,25 +79,30 @@ make build
...
+# Enable transaction statistics
 export ENABLE_TRANSACTION_STATS="true"
 export TRANSACTION_STATS_HISTORY_LENGTH_DAYS="366"
 export TRANSACTION_STATS_UPDATE_INTERVAL="3600"
+
+# Enable fee estimator
+export ENABLE_FEE_ESTIMATOR_QUICK="true"
 ```
 
+When running a local instance of Lisk Core and Lisk Service that is run by Docker, then the following variable needs to be set: `LISK_CORE_WS="ws://host.docker.internal:8080"`.
```
Comment (sameersubudhi) [inline README.md:104]:
```suggestion
When running a local instance of Lisk Core and Lisk Service that is run by Docker, the following variable needs to be set: `LISK_CORE_WS="ws://host.docker.internal:8080"`.
```

File: docs/build_from_source.md [hunk 3]
```diff
@@ -112,25 +112,37 @@ npm test
 
 Functional tests ensure that a project build on the top of the framework is able to process requests and collect responses involving the API gateway.
 
-In order to run them successfully, they need to have the template component running alongside.
+In order to run them successfully, it is necessary to have the template component running alongside.
```
Comment (sameersubudhi) [inline docs/build_from_source.md:115]:
```suggestion
In order to run them all successfully, it is necessary to have the template component running alongside.
```

File: README.md [hunk 4]
```diff
@@ -4,27 +4,30 @@
...
+| [Lisk](services/core) | The Lisk Core service acts as a bridge between the Lisk Core and the Lisk Service API. Its main purpose is to provide enriched data from the Lisk Core API. This service is aimed at providing high availability, and both efficient and reliable access to the Lisk Core API. |
+| [Market](services/market) | The Market service allows price data retrieval. It supports multiple sources to keep the current Lisk token price up-to-date and available to the clients. |
 | [Template](services/template) | The Template service is an abstract service that all of Lisk Service services are inherited from. It allows all services to share a similar interface and design pattern. Its purpose is to reduce code duplication and increase consistency between each service, hence simplifying code maintenance and testing. |
 
 **Remarks**
 
-- Lisk Service is configured to connect [Lisk mainnet network](https://explorer.lisk.io/) by default.
+- Lisk Service is configured to connect a local node on port 4000 by default.
```
Comment (sameersubudhi) [inline README.md:28]:
```suggestion
- Lisk Service is configured to connect a local node via WS on port 8080 by default.
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Kindly consider the following remarks.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**